Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/openshift/origin-release:golang-1.16 AS builder
+FROM docker.io/openshift/origin-release:golang-1.17 AS builder
```

Comment:
@TheRealHaoLiu which base image are we supposed to use for golang 1.17 do you think?